consensus/module.go

```diff
@@ -1,37 +1,40 @@
 package consensus
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
 
 	"github.com/pokt-network/pocket/consensus/leader_election"
-	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
 	typesCons "github.com/pokt-network/pocket/consensus/types"
 	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
-	"github.com/pokt-network/pocket/shared/modules"
-	"github.com/pokt-network/pocket/shared/types"
-	"github.com/pokt-network/pocket/shared/types/genesis"
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
+
+	consensusTelemetry "github.com/pokt-network/pocket/consensus/telemetry"
+	"github.com/pokt-network/pocket/shared/modules"
 )
 
 const (
 	DefaultLogPrefix string = "NODE" // Just a default that'll be replaced during consensus operations.
 )
 
+var _ modules.ConsensusGenesisState = &typesCons.ConsensusGenesisState{}
+var _ modules.PacemakerConfig = &typesCons.PacemakerConfig{}
+var _ modules.ConsensusConfig = &typesCons.ConsensusConfig{}
 var _ modules.ConsensusModule = &consensusModule{}
 
 // TODO(olshansky): Any reason to make all of these attributes local only (i.e. not exposed outside the struct)?
 type consensusModule struct {
 	bus        modules.Bus
 	privateKey cryptoPocket.Ed25519PrivateKey
-	consCfg    *genesis.ConsensusConfig
+	consCfg    modules.ConsensusConfig
```

Comment:
Can we access this via the bus and not need to saved an extra reference to it?

Reply:
You can access the raw json in the bus

Reply:
`GetConfig() map[string]json.RawMessage`